pkgs/smallstep/cli/scaffold.yaml

```diff
@@ -0,0 +1,3 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/name: smallstep/cli
```

Comment:
🤔 Looks like the comment is broken.

Reply:
I'm not sure why this was broken.

https://github.com/aquaproj/aqua/blob/88078fcd0aa9755e6b90ebe14a343dc0a43bf506/pkg/controller/generate-registry/init.go#L13

Reply:
Did you rebase the commit?

Reply:
Ah, I see.
`cmdx s` doesn't create this file.
`cmdx s` simply copies a file.
So this file can be broken without rebasing.

Reply:
Yeah, I probably broke it locally after scaffolding and testing, while cleaning up some generated example comments I presume are not needed here.